packages/component/src/Activity2/Avatar.js

```diff
@@ -0,0 +1,17 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { withStyleSet } from '../Context';
+
+export default withStyleSet(({
+  children,
+  className,
+  styleSet
+}) =>
+  <div className={ classNames(
+    styleSet.avatar + '',
+    (className || '') + '')
+  }>
+    { children }
+  </div>
```

Comment:
adds a div around... do we want that or to augment the existing control with additional style classes?

Reply:
Will keep `<div>`, unlike font icon or similar.

The `<div>` is a circle clip and the children can be either initials or image. So it more like an actual component although it is small in size.